test.properties

```diff
@@ -3211,6 +3253,292 @@
             (testray.main.component.name == "User Tracker")\
         )
 
+    #
+    # Core Infrastructure Acceptance
+    #
+
+    modules.includes[modules-semantic-versioning][core-acceptance]=${core.infra.testing.modules}
+
+    modules.includes.public[modules-semantic-versioning][core-acceptance]=${core.infra.testing.modules}
+
+    test.batch.class.names.includes[modules-integration-aws-s3-mysql57][core-acceptance]=\
+        **/apps/change-tracking/change-tracking-store-test/**/*Test.java,\
+        **/apps/portal-store/**/*Test.java
+
+    test.batch.class.names.includes[modules-integration-*][core-acceptance]=\
+        **/apps/bean-portlet/**/*Test.java,\
+        **/apps/change-tracking/change-tracking-store-test/**/*Test.java,\
```

Comment:
Remove **/apps/change-tracking/change-tracking-store-test/**/*Test.java, not us

Reply:
For this one, I added because these tests are used for the "Document Library Stores".